### ISSUE TITLE
[stable/openebs] Missing configuration for prometheus scrape

### ISSUE THREAD

**Issue by pavloneiman** [helm/charts#9691]
**FEATURE REQUEST**:

Monitoring is enabled by default and there are parameters in values.yaml that allow to set image and tag for Prometheus Exporter (https://github.com/helm/charts/tree/master/stable/openebs) but there are no way how to set annotations "prometheus.io/scrape" and "prometheus.io/port".

**values.yaml:** [charts/stable/openebs/values.yaml](https://github.com/helm/charts/blob/a559e0a27dcbee1475c0ecae7992d1cd930dbab5/stable/openebs/values.yaml)
```
...
policies:
  monitoring:
    enabled: true
    image: "quay.io/openebs/m-exporter"
    imageTag: "0.7.2"
```

**Version of Helm and Kubernetes**: v2.10.0 and v1.12.3
**Which chart**:  stable/openebs
**What you expected to happen**: way to configure prometheus scrape

**Comment by kmova** [comment]
Will be fixed with 0.8.0 - https://github.com/openebs/maya/pull/857

**Comment by kmova** [comment]
The chart has been updated to 0.8.0. If the monitoring is enabled, the "prometheus.io/scrape" and "prometheus.io/port" annotations are added to the volume target pods.
